ms-common-impl/src/main/java/net/trajano/ms/vertx/VertxMicroserviceEngine.java

```diff
@@ -89,21 +52,35 @@ public static void run(final Class<? extends Application> applicationClass,
     private VertxOptions vertxOptions;
 
     /**
-     * Prevent instantiation.
+     * Sets the system properties and sets up the logger. {@inheritDoc}
      */
-    private Microservice() {
+    @Override
+    public Object[] bootstrap() {
+
+        System.setProperty("vertx.disableDnsResolver", "true");
+        System.setProperty("vertx.logger-delegate-factory-class-name", "io.vertx.core.logging.SLF4JLogDelegateFactory");
+
+        final File logbackFile = new File("logback.xml");
+        if (logbackFile.exists()) {
+            System.setProperty("logging.config", logbackFile.getAbsolutePath());
+        }
 
+        final Object[] sources = new Object[] {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "sources". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)